thumbor/engines/pil.py

```diff
@@ -388,3 +388,6 @@ def strip_icc(self):
 
     def strip_exif(self):
         self.exif = None
+
+    def blur(self, radius):
+        self.image = self.image.filter(ImageFilter.GaussianBlur(radius))
```

Comment:
This method has to be defined in parent class, otherwise it's unusable with non-pillow engines.
I am not sure what we could do about BC break in this case. 
Maybe we could use blur from filters? It works on raw RGB data.